src/components/2-Monitoring/Chapter2Icons2.vue

```diff
@@ -87,90 +87,83 @@
 </script>
 
 <style scoped lang="scss">
-    #icons {
-      min-width: 375px;
-      max-width: 700px;
-      margin: 0 auto;
-      padding-right: 2em;   
-      @media screen and (max-width: 600px) {
-          padding-right: 25px;
-      }
-    }
-    .grid-container {
-        padding: 10px;
-        padding-bottom: 20px;
-        display: grid;
-        grid-template-columns: 33.33% 33.33% 33.33%;
-        grid-template-rows: 100%;
-        gap: 0px 5px;
-        grid-template-areas:
-            "icon-1 icon-2 icon-3";
-        min-width: 0;
-        min-height: 0;
-        @media screen and (max-width: 600px) {
-          padding: 10px 0;
-          gap: 0px;
-        }
-    }
 
-    .grid-item {
-        padding: 5px;
-    }
+// IMPORT COLORS
+$backgroundCharcoal: #202226; //#1E1F23 #26282b 
+$boxCharcoal: #171717; //#28292D #1a1b1f
+$offWhite: #F1F1F1;
+$plasmaYellow: #FAB62F;
+$plasmaPink: #BE3D7D;
+$plasmaPurple: #62039A;
+$plasmaBlue: #142167;
+$darkBlue: #0F2237;
+$footerBlue: #00264C;
+$mediumBlue: #5191bd; //#63B1E6 #579ecf
+$lightBlue: #AAD1EC;
+$grayBlue: #4F5C67;
 
-    .icon-item{
-        text-align: center;
-        margin: auto;
+#icons {
+  min-width: 375px;
+  max-width: 700px;
+  margin: 0 auto;
+  padding-right: 2em;   
+  @media screen and (max-width: 600px) {
+      padding-right: 25px;
+  }
+}
+.grid-container {
+    padding: 10px;
+    padding-bottom: 20px;
+    display: grid;
+    grid-template-columns: 33.33% 33.33% 33.33%;
```

Comment:
In this case you can also use `1fr 1fr 1fr` to declare grid dimensions in fractions of "remaining available space" (e.g. `1 fraction`). That way the grid is sized to the parent element and takes into account any padding within the `.grid-container`.

Reply:
Aha -- I was wondering exactly how that `1fr` piece works. I'll try that now